framework/mail/Template.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\mail;
+
+use Yii;
+use yii\base\Object;
+use yii\base\ViewContextInterface;
+
+/**
+ * Template composes the message from view templates, ensuring isolated view rendering. It allows
+ * changing of the rendering options such as layout during composing of the particular message without
+ * affecting other messages.
+ *
+ * An instance of this class serves as a view context during the mail template rendering and is available
+ * inside a view template file via [[\yii\base\View::context]].
+ *
+ * @see BaseMailer::compose()
+ *
+ * @author Paul Klimov <klimov.paul@gmail.com>
+ * @since 2.1
+ */
+class Template extends Object implements ViewContextInterface
+{
+    /**
+     * @var MessageInterface related mail message instance.
+     */
+    public $message;
+    /**
+     * @var \yii\base\View view instance used for rendering.
+     */
+    public $view;
+    /**
+     * @var string path to the directory containing view files.
+     */
+    public $viewPath;
+    /**
+     * @var string|array name of the view to use as a template. The value could be:
+     *
+     * - a string that contains either a view name or a path alias for rendering HTML body of the email.
```

Comment:
Maybe use references to similar Composer's properties to avoid descriptions' repetition?